### PR TITLE
chore: use simplehash for polygon nfts

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-4443 RNBW-4545
Figma link (if any):

## What changed (plus any additional context for devs)
now we use simplehash for polygon nft fetching.

## Screen recordings / screenshots

PoW: https://cloud.skylarbarrera.com/Screen-Recording-2022-09-28-11-20-14.mp4

## What to test

allowlisted polygon nfts should still show up, should have no impact on arbitrum or optimism or mainnet nfts.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by skylarbarrera** [comment]
@jxom @estebanmino this PR is ready for review

**Review by BrodyHughes — APPROVED** [review]
Tested: 

- Mainnet
- POAPs
- Optimism NFTs
- Arbitrum NFTs
- Polygon NFts
- Spot checked a couple random Polygon items from our allow list

All looks good!

File: src/components/expanded-state/UniqueTokenExpandedState.tsx [hunk 1]
```diff
@@ -229,7 +229,6 @@ interface UniqueTokenExpandedStateProps {
 const getIsSupportedOnRainbowWeb = (network: Network) => {
   switch (network) {
     case Network.mainnet:
-    case Network.polygon:
```
Comment (estebanmino) [inline src/components/expanded-state/UniqueTokenExpandedState.tsx:232]: would be nice to add a comment here to re enable when allow list is added to web as well

**Review by estebanmino — APPROVED** [review]
LGTM, i didn't dev qa since it's already qaed

**Comment by osdnk** [comment]
Android screenshots?

**Comment by skylarbarrera** [comment]
> Android screenshots?

this is just logic, no UI changes, shouldn't need android screens for this

**Pull request merged**